backend/controllers/users.js

```diff
@@ -8,6 +10,8 @@ const {
   Lesson_word,
 } = require("../models");
 
+const salt = 11;
```

Comment:
- what is this constant variable for?

Reply:
The salt variable is used in bcrypt when hashing a password.